src/cljs/rems/navbar.cljs

```diff
@@ -60,18 +60,20 @@
 
 (defn navbar-items [e page-id identity]
   ;;TODO: get navigation options from subscription
-  (let [roles (:roles identity)]
+  (let [roles (:roles identity)
+        config @(rf/subscribe [:rems.config/config])
+        catalogue-is-public (when config (config :catalogue-is-public))]
```

Comment:
Does there need to be a `when` here?

Reply:
I find it logical?

Reply:
If `config` is `nil`, `(config :catalogue-is-public)` is a null pointer exception. However, if you use `(:catalogue-is-public config)` you get `nil` when `config` is `nil`, giving you the same behaviour as this code. Not insisting you change it, just pointing this out.

Reply:
Yes I'd prefer `(:catalogue-is-public config)`, no need for `when`.